exec/cnex/lib/sys.h

```diff
@@ -0,0 +1,12 @@
+#ifndef SYS_H
+#define SYS_H
+
+#include "cell.h"
```

Comment:
Use forward declaration of `Cell` instead of include.

Reply:
Fixed.